bylaws.md

```diff
@@ -577,26 +577,52 @@ The corporation has issued no shares of stock, and has no shareholders.
 
 We, the undersigned, are all of the persons acting as the initial directors of Django Events Foundation North America, a California nonprofit corporation, and, pursuant to the authority granted to the directors by these bylaws to take action by unanimous written consent without a meeting, consent to, and hereby do, adopt the foregoing bylaws, consisting of 35 pages, as the bylaws of this corporation.
 
-Dated: Jan 2, 2019
+Dated: April 9, 2021
 
 
-Jeffrey Triplett, Director
 
-Monique Murphy, Director
+____________________
+Josue Balandrano Coronel, Director
 
-Katia Lira, Director
 
+____________________
 Adam Fast, Director	
 
+
+____________________
+Carol Ganz, Director
+
+
+____________________
+Katia Lira, Director
+
+
+____________________
 Katherine Michel, Director
 
-Josue Balandrano Coronel, Director
+
+____________________
+Monique Murphy, Director
+
+
+____________________
+Jen Myers, Director
+
+
+____________________
+Jeffrey Triplett, Director
+
+
+____________________
+Drew Winstel, Director
```

Comment:
Do you need Robert Andrew Winstel since Drew is not my first name?

Reply:
It's up to you. I usually put whatever is the most formal name I've been given.

Reply:
Let's go ahead and make the switch then.

Reply:
Sounds good. I went ahead and did it.